plugin/pkg/scheduler/core/extender.go

```diff
@@ -193,6 +195,27 @@ func (h *HTTPExtender) Prioritize(pod *v1.Pod, nodes []*v1.Node) (*schedulerapi.
 	return &result, h.weight, nil
 }
 
+// Bind delegates the action of binding a pod to a node to the extender.
+func (h *HTTPExtender) Bind(binding *v1.Binding) error {
+	var result schedulerapi.BindingResult
+	if h.bindVerb == "" {
+		return nil
```

Comment:
Yes, its an artifact from earlier implementation. It shouldn't happen, but I will leave the safeguard.